site/docs/.vuepress/configs/en.ts

```diff
@@ -345,6 +345,10 @@ export const localeEn: LocaleConfig<DefaultThemeLocaleData> = {
           {
             text: "Tutorials",
             children: [
+              {
+                text: "Cloudflare Workers",
+                link: "/hosting/cloudflare-workers.html",
+              },
```

Comment:
I think it would be more fitting to list this entry below the Supabase Edge Function or the Deno Deploy. This is just a personal preference to prioritize Deno and the open-source option among the other entries.

Reply:
They are meant to be sorted A-Z. @KnorpelSenf correct me if I am wrong.

Reply:
I agree with @CikiMomogi.